Teaching/Github-Actions/Github-Actions.md

```diff
@@ -0,0 +1,47 @@
+# Github Actions
+
+## Story
+
+In software development, it's essential to maintain a structured workflow to manage changes efficiently and ensure code quality. Git workflows like GitFlow and GitHub Flow provide guidelines for managing branching strategies and integrating changes smoothly.
```

Comment:
It would be good to give more background here. Explaining what CI/CD is and a bit about github actions too. Also, what is Github Flow?

Reply:
Github flow is the basic workflow, like it includes creating branch, making changes, sending PR then merging